### PR TITLE
KOGITO-9459 Review Dev UI extension guide

### PR DESCRIPTION
**Pull request by paulovmr**
**JIRA:** [KOGITO-9459](https://issues.redhat.com/browse/KOGITO-9459)

**Description:** Reviews Dev UI extension guide, including update on the diagram images and the inclusion of the Monitoring page.

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

File: serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/quarkus-dev-ui-extension/quarkus-dev-ui-monitoring-page.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+* Amount of workflows on each status.
+* Average time spent on each workflow state.
+* List of workflows executed and their details.
+
+== Viewing detailed information about the execution of workflow instances
+To view detailed information about a specific workflow instance execution, you can use the *Workflows* tab. There you
+can find details such as when the workflow was started, its state and how much time its execution took. Also, you can
+check how much time was spent on each state.
+
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/quarkus-dev-ui-extension/quarkus-dev-ui-monitoring-page.adoc:23]:
```suggestion
To view detailed information about a specific workflow instance execution, you can use the *Workflows* tab. In the *Workflows* tab, you can find details such as when the workflow started, its state, and how long its execution took. Also, you can check how much time was spent on each state.
```

**Review by kaldesai — COMMENTED** [review]
@paulovmr Just a small suggestion. Otherwise, your content looks good to me. Thank you.

**Review by radtriste — APPROVED** [review]

File: serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/quarkus-dev-ui-extension/quarkus-dev-ui-monitoring-page.adoc [hunk 2]
```diff
@@ -0,0 +1,27 @@
+= Monitoring in Kogito Serverless Workflow Tools extension
+:compat-mode!:
+// Metadata:
+:description: Monitoring in Kogito Serverless Workflow Tools extension
+:keywords: kogito, workflow, serverless, Quarkus, Dev UI, monitoring
+
+In the Kogito Serverless Workflow Tools extension, the *Monitoring* page displays default dashboards that might be useful
```
Comment (ricardozanini) [inline serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/quarkus-dev-ui-extension/quarkus-dev-ui-monitoring-page.adoc:7]:
```suggestion
In the {product_name} Tools extension, the *Monitoring* page displays default dashboards that might be useful
```
Can you use this variable instead? https://github.com/kiegroup/kogito-docs/blob/main/serverlessworkflow/antora.yml#L13

**Review by ricardozanini — COMMENTED** [review]
Not sure if we are not using the `{product_name}` variable in other parts of the tooling guide. Can you review?

File: serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/quarkus-dev-ui-extension/quarkus-dev-ui-monitoring-page.adoc (hunk 1 above)
Comment (paulovmr) [reply serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/quarkus-dev-ui-extension/quarkus-dev-ui-monitoring-page.adoc:20]: Fixed, thanks.

**Review by ricardozanini — APPROVED** [review]

**Review by ederign — APPROVED** [review]

**Review by domhanak — APPROVED** [review]
LGTM

**Comment by paulovmr** [comment]
@kaldesai if you could please review so we can merge. Thanks!

**Pull request merged**